doc/man3/SSL_read_early_data.pod

```diff
@@ -267,6 +267,19 @@ Nagle's algorithm. If an application opts to disable Nagle's algorithm
 consideration should be given to turning it back on again after the handshake is
 complete if appropriate.
 
+In rare circumstances, it may be possible for a client to have a session that
+reports a max early data value greater than 0, but where the server does not
+support this. For example, this can occur if a server has had its configuration
+changed to accept a lower max early data value such as by calling
+SSL_CTX_set_recv_max_early_data(). Another example is if a server used to
+support TLSv1.3 but was later downgraded to TLSv1.2. Sending early data to such
+a server will cause the connection to abort. Clients that encounter an aborted
+connection while sending early data may choose to retry the connection without
+sending early data. This does not happen automatically. A client will have to
+establish a new transport layer connection to the server and attempt the SSL/TLS
+connection again but without sending early data. Note that it is inadvisable to
```

Comment:
s/is/might be/

Reply:
Why only "might be"? Under what conditions would we advise it?

Reply:
Suppose the server got a new revision that included TLS 1.3 support, but some of the new features caused it to crash.

Reply:
that's a bug in that server and workarounding it should be applied either to none or all handshakes

Reply:
I'm not sure I correctly understand the senes of Rich's discomfort with this text.  FWIW, I would be happy to replace "inadvisable" with "forbidden by the spec".